design-tokens/tokens/layer1/base.json

```diff
@@ -113,78 +125,150 @@
       "type": "letterSpacing"
     },
     "increased": {
-      "value": "150%",
+      "value": "2%",
       "type": "letterSpacing"
     },
     "decreased": {
-      "value": "-5%",
+      "value": "-2%",
       "type": "letterSpacing"
     }
   },
   "paragraphSpacing": {
-    "h1": {
-      "value": "32",
+    "small": {
+      "value": "1rem",
+      "type": "paragraphSpacing"
+    },
+    "medium": {
+      "value": "1.125rem",
       "type": "paragraphSpacing"
     },
-    "h2": {
-      "value": "26",
+    "large": {
+      "value": "1.375rem",
       "type": "paragraphSpacing"
     }
   },
   "fontWeights": {
-    "headingRegular": {
-      "value": "Regular",
-      "type": "fontWeights"
-    },
-    "headingBold": {
+    "bold": {
       "value": "Bold",
       "type": "fontWeights"
     },
-    "bodyRegular": {
-      "value": "Regular",
+    "semiBold": {
+      "value": "Semi Bold",
       "type": "fontWeights"
     },
-    "bodyBold": {
-      "value": "Bold",
+    "regular": {
+      "value": "Regular",
       "type": "fontWeights"
     }
   },
   "fontSizes": {
-    "h1": {
-      "value": "{fontSizes.h2} * 1.25",
+    "headingXXXL": {
+      "value": "2.125rem",
       "type": "fontSizes"
     },
-    "h2": {
-      "value": "{fontSizes.h3} * 1.25",
+    "headingXXL": {
+      "value": "1.5rem",
       "type": "fontSizes"
     },
-    "h3": {
-      "value": "{fontSizes.h4} * 1.25",
+    "headingXL": {
+      "value": "1.25rem",
       "type": "fontSizes"
     },
-    "h4": {
-      "value": "{fontSizes.h5} * 1.25",
+    "headingL": {
+      "value": "1rem",
       "type": "fontSizes"
     },
-    "h5": {
-      "value": "{fontSizes.h6} * 1.25",
+    "headingM": {
+      "value": "0.85rem",
       "type": "fontSizes"
     },
-    "h6": {
-      "value": "{fontSizes.body} * 1",
+    "headingS": {
+      "value": "0.75rem",
       "type": "fontSizes"
     },
-    "body": {
-      "value": "16",
+    "bodyLarge": {
+      "value": "1rem",
       "type": "fontSizes"
```

Comment:
Maybe a stupid question, but shouldn't `1 rem` be the "normal" size?

Reply:
Agree, that's also currently the case in MIUI

Reply:
16 is the base that we used to calculate the type scale. Is the boundary between a text and a heading. With 16 you can go up and down, to find other font sizes.

"Default" or "normal" means that is the most used value, the go-to font size and that value is 14px. 
- If you need to display relevant information, you use Default. 
- If you need less relevant info, like metadata, you use the small.
- If you need to display impactful information, you go for the large.

Reply:
And actually when looking at the current UI most of the text is 14px

Reply:
True :)